src/index.html

```diff
@@ -16,17 +16,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="position">
+    <form
+      action="#"
+      data-qa="big"
+      class="form form__big"
```

Comment:
The class name 'form__big' seems to represent the style or size of the form rather than its content. Class names should represent the meaning of the content, not the styles or tag names.